### PR TITLE
PMM-4177 Full PostgreSQL pgstatements Agent support.

### PR DESCRIPTION
**Pull request by BupycHuk**

### CODE REVIEW

File: models/data_ingestion.go [hunk 1]
```diff
@@ -367,7 +382,22 @@ const insertSQL = `
...
+  	:m_local_blks_cnt,
+  	:m_local_blks_hit,
+  	:m_local_blks_read,
+  	:m_local_blks_dirtied,
+  	:m_local_blks_written,
+  	:m_temp_blks_cnt,
+  	:m_temp_blks_read,
+  	:m_temp_blks_written,
+  	:m_blk_read_time,
+  	:m_blk_write_time
```
Comment (askomorokhov) [inline models/data_ingestion.go]: Please align this query

File: models/data_ingestion.go [hunk 2]
```diff
@@ -197,7 +197,22 @@ const insertSQL = `
...
+  	m_local_blks_cnt,
+  	m_local_blks_hit,
+  	m_local_blks_read,
+  	m_local_blks_dirtied,
+  	m_local_blks_written,
+  	m_temp_blks_cnt,
+  	m_temp_blks_read,
+  	m_temp_blks_written,
+  	m_blk_read_time,
+  	m_blk_write_time
```
Comment (askomorokhov) [inline models/data_ingestion.go]: Following the call with @BupycHuk, PostgreSQL metrics should be stored in next way:

```suggestion
  	  	m_shared_blks_hit_cnt,
	  	m_shared_blks_hit_sum,
	  	m_shared_blks_read_cnt,
	  	m_shared_blks_read_sum,
	  	m_shared_blks_dirtied_cnt,
	  	m_shared_blks_dirtied_sum,
	  	m_shared_blks_written_cnt,
	  	m_shared_blks_written_sum,
	  	m_local_blks_hit_cnt,
	  	m_local_blks_hit_sum,
	  	m_local_blks_read_cnt,
	  	m_local_blks_read_sum,
	  	m_local_blks_dirtied_cnt,
	  	m_local_blks_dirtied_sum,
	  	m_local_blks_written_cnt,
	  	m_local_blks_written_sum,
	  	m_temp_blks_read_cnt,
	  	m_temp_blks_read_sum,
	  	m_temp_blks_written_cnt,
	  	m_temp_blks_written_sum,
	  	m_blk_read_time_cnt,
	  	m_blk_read_time_sum,
	  	m_blk_write_time_cnt,
	  	m_blk_write_time_sum
```

Also we need to refactor other related code.

File: models/metrics.go [hunk 3]
```diff
@@ -318,7 +336,19 @@ if(SUM(m_no_index_used_cnt) == 0, NaN, SUM(m_no_index_used_sum) / time_frame) AS
...
+if(SUM(m_shared_blks_cnt) == 0, NaN, SUM(m_shared_blks_dirtied) / time_frame) AS m_shared_blks_dirtied_per_sec,
+if(SUM(m_shared_blks_cnt) == 0, NaN, SUM(m_shared_blks_written) / time_frame) AS m_shared_blks_written_per_sec,
+if(SUM(m_local_blks_cnt) == 0, NaN, SUM(m_local_blks_hit) / time_frame) AS m_local_blks_hit_per_sec,
+if(SUM(m_local_blks_cnt) == 0, NaN, SUM(m_local_blks_read) / time_frame) AS m_local_blks_read_per_sec,
+if(SUM(m_local_blks_cnt) == 0, NaN, SUM(m_local_blks_dirtied) / time_frame) AS m_local_blks_dirtied_per_sec,
+if(SUM(m_local_blks_cnt) == 0, NaN, SUM(m_local_blks_written) / time_frame) AS m_local_blks_written_per_sec,
+if(SUM(m_temp_blks_cnt) == 0, NaN, SUM(m_temp_blks_read) / time_frame) AS m_temp_blks_read_per_sec,
+if(SUM(m_temp_blks_cnt) == 0, NaN, SUM(m_temp_blks_written) / time_frame) AS m_temp_blks_written_per_sec,
+if(SUM(m_temp_blks_cnt) == 0, NaN, SUM(m_blk_read_time) / time_frame) AS m_blk_read_time_per_sec,
+if(SUM(m_temp_blks_cnt) == 0, NaN, SUM(m_blk_write_time) / time_frame) AS m_blk_write_time_per_sec
```
Comment (askomorokhov) [inline models/metrics.go]: Please change `_per_sec` suffix to `_sum_per_sec`.

**Comment by AlekSi** [comment]
I will resolve conflicts

**Pull request merged**